cypress/support/e2e.ts

```diff
@@ -16,8 +16,9 @@
 import "@cypress/code-coverage/support";
 import "./commands";
 
-Cypress.Server.defaults({
-  ignore: (xhr: Request) => {
-    return xhr.url.match(/^.+\/v0\/playground\/$/);
-  },
-});
+// to rewrite to be able to support in Cypress 12
+// Cypress.Server.defaults({
+//   ignore: (xhr: Request) => {
+//     return xhr.url.match(/^.+\/v0\/playground\/$/);
+//   },
+// });
```

Comment:
Just to clarify, what is the impact of us commenting out this piece ?

Reply:
I haven't found any, not sure if that part of the code used anywhere. 
Can be deleted optionally.